lib/commands/context.js

```diff
@@ -117,12 +122,27 @@ commands.mobileGetContexts = async function mobileGetContexts () {
   return await webviewHelpers.getWebViewsMapping(this.adb, opts);
 };
 
-helpers.switchContext = async function switchContext (name) {
+helpers.getContextsWithWebviewMappings = function getContexts (webViewsMapping) {
```

Comment:
please align the variable and the function names

Reply:
I assume the function must be async, because webviewHelpers.getWebviews is async

Reply:
>  please align the variable and the function names

Addressed in https://github.com/appium/appium-android-driver/pull/699/commits/a83c7b256e03c645c445ad680e09d358e0ffa026

> I assume the function must be async, because webviewHelpers.getWebviews is async

I'm afraid, but `webviewHelpers.getWebviews` is not async now.